OneCommunityGlobal/HighestGoodNetworkApp
# Samarth B Job CC feature

# Description
Create a UI for an Owner to add to the CC options for specific posting categories, jobs, and/or ALL positions - Create a modal or management dashboard for the owner to view and manage the CC lists.
Should show whomever is on this list
Make it so that the CC list checks to make sure the person is active, and removes them from the list if they aren’t
Have CC functionality to add other people to CC so people who are hiring for a particular position can get a CC of the application details
Allow the owner to assign CC permissions: At the category level (example all ads under “Software Engineer”), For individual ads within a category (example only Front-End Engineer ads).


## Related PRS (if any):
This frontend PR is related to the [#1165](https://github.com/OneCommunityGlobal/HGNRest/pull/1165) backend PR.
…

## How to test:
1. check into current branch and follow backend PR instructions
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to Dashboard
6. "Edit Job Application Email CC" button should be visible below "Blue Square Email CC" button
7. Clicking the button should redirect you to /job-notification-dashboard
8. Verify Adding Emails to Particular job posting as well as addcing CC to each category work.

## Screenshots or videos of changes:


https://github.com/user-attachments/assets/30915b7a-afcb-4d60-98b0-3c01b8ccef29
